From: John Salatas <jsalatas@gmail.com>
Date: Sun, 18 Jun 2017 11:44:34 -0700
Subject: Updated check copyright test

---
 tests/copyright/check_copyright.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/copyright/check_copyright.sh b/tests/copyright/check_copyright.sh
index 0751f58..ca7e33b 100755
--- a/tests/copyright/check_copyright.sh
+++ b/tests/copyright/check_copyright.sh
@@ -48,7 +48,7 @@ ignore_pat="$ignore_pat|/tests/.*/.*\\.desktop$"
 # xml files
 ignore_pat="$ignore_pat|\\.xml$"
 # /data/ files
-ignore_pat="$ignore_pat|/data/.*\\.conf$|/data/.*\\.in|/data/.*\\.url-dispatcher$|/data/.*\\.pkla$"
+ignore_pat="$ignore_pat|/data/.*\\.conf$|/data/.*\\.in|/data/.*\\.url-dispatcher$|/data/.*\\.pkla|/data/lightdm-session$"
 # polkit files
 ignore_pat="$ignore_pat|\\.pkla$"
 # /debian/ files
